be/src/runtime/file_result_writer.cpp

```diff
@@ -216,12 +218,82 @@ std::string FileResultWriter::_file_format_to_name() {
         return "unknown";
     }
 }
+std::string FileResultWriter::gen_types() {
+    std::string types = "";
+    int num_columns = _output_expr_ctxs.size();
+    for (int i = 0; i < num_columns; ++i) {
+        switch (_output_expr_ctxs[i]->root()->type().type) {
+        case TYPE_BOOLEAN:
+        case TYPE_TINYINT:
+            types += "TINYINT";
```

Comment:
1. can we use type_to_string in primitive_type.cpp?
2. the string for types, it is better the same as types name in create table SQL?

Reply:
ok, it's a good idea